app/assets/stylesheets/config/_variables.scss

```diff
@@ -52,6 +52,13 @@
   --transition-time: 100ms;
   --transition-props: var(--transition-func) var(--transition-time);
 
+  ////////////////////////////////////////////////////////////////////////////////////
+  // Focus ring //////////////////////////////////////////////////////////////////////
+  ////////////////////////////////////////////////////////////////////////////////////
+
+  --focus-ring: 0 0 0 2px var(--base-inverted),
+    0 0 0 4px var(--button-primary-bg-hover); // TODO: replace with appropriate variable from _colors.scss
```

Comment:
yes, as I wanted to have a dedicated _colors.scss variable for that ring color only. It will likely end up being the same colors but I wanna be sure naming is not confusing.

Reply:
I guess we still need to introduce this variable?

Reply:
Yes, but not now. It should be some more theming-oriented PR.